### PR TITLE
Upgrade of samples

### PR DESCRIPTION
**Pull request by piotrmilcarz**

### CODE REVIEW

File: Electron/Electron12/protected/package.json [hunk 1]
```diff
@@ -68,6 +66,8 @@
     "@electron-forge/maker-zip": "~6.0.0-beta.54",
     "@electron-forge/plugin-webpack": "6.0.0-beta.54",
     "@marshallofsound/webpack-asset-relocator-loader": "~0.5.0",
+    "@preemptive/jsdefender-core": "file:assets/preemptive-jsdefender-core-2.6.0.tgz",
+    "@preemptive/jsdefender-webpack-plugin": "file:assets/preemptive-jsdefender-webpack-plugin-2.6.0.tgz",
```
Comment (tofir) [inline Electron/Electron12/protected/package.json:70]: please revert this change, it's left for edit intentionally

File: Electron/Electron12/protected/jsdefender.config.json [hunk 2]
```diff
@@ -30,7 +30,7 @@
         },
         "propertyIndirection": true,
         "selfDefending": true,
-        "stringLiterals": true,
+        "stringLiterals": false,
```
Comment (tofir) [inline Electron/Electron12/protected/jsdefender.config.json:33]: why is this turned off? do we have issue with string literals enabled?

**Comment by tofir** [comment]
@piotrmilcarz left few notes
cc @speedywizard

**Review by tofir — CHANGES_REQUESTED** [review]

File: Electron/Electron12/protected/jsdefender.config.json (hunk 2 above)
Comment (piotrmilcarz) [reply Electron/Electron12/protected/jsdefender.config.json:33]: When stringLiterals is enabled then css definitions are broken.
It happens in both versions of jsdefender: official/current version 2.6.0 and in new one.

File: Electron/Electron12/protected/package.json (hunk 1 above)
Comment (piotrmilcarz) [reply Electron/Electron12/protected/package.json]: 👍

File: Angular/Angular15/.DS_Store
Comment (tofir) [inline Angular/Angular15/.DS_Store:1]: please remove this one

File: Electron/Electron12/protected/jsdefender.config.json (hunk 2 above)
Comment (tofir) [reply Electron/Electron12/protected/jsdefender.config.json]: basically, you can simply remove desired files from obfuscation

**Review by tofir — CHANGES_REQUESTED** [review]
@piotrmilcarz please see my notes
considering apps build fine - in the rest it looks good
